# HG changeset patch
# User Ben Abbott <bpabbott@mac.com>
# Date 1236632863 14400
#      Mon Mar 09 17:07:43 2009 -0400
# Node ID 4a312440b262d4576c8102a4b9aac475d5d79046
# Parent  c4383701e10d78f032e09375153f1f26b4c0e7e3
__go_draw_axes__.m (do_tics): fix typo

diff --git a/scripts/ChangeLog b/scripts/ChangeLog
--- a/scripts/ChangeLog
+++ b/scripts/ChangeLog
@@ -1,8 +1,13 @@
+2009-03-09  Ben Abbott  <bpabbott@mac.com>
+
+	* plot/__go_draw_axes__.m (do_tics): Fix typo (xaxislocation ->
+	yaxislocation).
+
 2009-03-09  John W. Eaton  <jwe@octave.org>
 
 	* help/lookfor.m (search_cache): Also match function names.
 
 	* pkg/pkg.m, help/gen_doc_cache.m, help/lookfor.m:
 	Use doc-cache instead of DOC for doc cache file.
 
 2009-03-09  Jaroslav Hajek  <highegg@gmail.com>
@@ -10,78 +15,78 @@ 2009-03-09  Jaroslav Hajek  <highegg@gma
 	* general/accumarray.m: Reorder tests. Call either "sparse" or
 	__accumarray_sum__ for the default summation case.
 	* statistics/base/histc.m: Reimplement using lookup & accumarray.
 
 2009-03-08  Søren Hauberg <hauberg@gmail.com>
 
 	* statistics/base/histc.m: New function.
 
-2009-03-06  Ben Abbott <bpabbott@mac.com>
+2009-03-06  Ben Abbott  <bpabbott@mac.com>
 
 	* plot/__go_draw_axes__.m: Preserve the order of axes' children
 	when pruning the handles for the {x,y,z}labels and title.
 	* plot/legend.m: Change demo to add colors to lines. This change
 	reveals the problem (above) with the ordering of the axes' children.
 
 2009-03-05  Jaroslav Hajek  <highegg@gmail.com>
 
 	* linear-algebra/condest.m: Use lu rather than splu. Use relative tols
 	for tests.
 
 2009-03-05  John W. Eaton  <jwe@octave.org>
 
 	* deprecated/spdiag.m: Ensure sparse result.
 
-2009-03-05  Ben Abbott <bpabbott@mac.com>
+2009-03-05  Ben Abbott  <bpabbott@mac.com>
 
 	* plot/__go_draw_axes__.m: Preserve column vector orientation for
 	the axes children while removing the axis label & title handles.
 
 2009-03-03  Jaroslav Hajek  <highegg@gmail.com>
 
 	* polynomial/polyval.m: Implement using Horner scheme.
 
-2009-03-03  Ben Abbott <bpabbott@mac.com>
+2009-03-03  Ben Abbott  <bpabbott@mac.com>
 
 	* plot/gnuplot_drawnow.m: Fix unintended shift of plot image for
 	the gnuplot postscript+eps terminal.
 
 2009-03-02  Jaroslav Hajek  <highegg@gmail.com>
 
 	* pkg/pkg.m (fix_depends): Fix & simplify splitting the string.
 
 2009-03-02  Thorsten Meyer  <thorsten.meyier@gmx.de>
 
 	* set/complement.m, set/union.m: Remove reference to deprecated
 	function create_set.
 
-2009-03-01  Ben Abbott <bpabbott@mac.com>
+2009-03-01  Ben Abbott  <bpabbott@mac.com>
 
 	* plot/__go_draw_axes__.m: Do not render axis labels twice when
 	their handles are visible, but do render objects with hidden
 	handles.
 
-2009-03-01  Ben Abbott <bpabbott@mac.com>
+2009-03-01  Ben Abbott  <bpabbott@mac.com>
 
 	* general/num2str.m: Trivial bug fix. Recent switch from split()
 	to strsplit() produced cells rather than character data.
 
-2009-03-01  Ben Abbott <bpabbott@mac.com>
+2009-03-01  Ben Abbott  <bpabbott@mac.com>
 
 	* plot/colorbar.m: Bug fix. Allow hidden colorbars to be deleted,
 	and replace existing colorbar when a new one is created. Additional
 	demos are included to verify these behaviors.
 
-2009-02-28  Ben Abbott <bpabbott@mac.com>
+2009-02-28  Ben Abbott  <bpabbott@mac.com>
 
 	* plot/__actual_axis_position__.m: Include file missed in prior
 	changeset.
 
-2009-02-28  Ben Abbott <bpabbott@mac.com>
+2009-02-28  Ben Abbott  <bpabbott@mac.com>
 
 	* plot/print.m, plot/gnuplot_drawnow.m: Add compatible support for
 	specificying resolution of bitmap terminals, "-r<num>".
 
 	* plot/gnuplot_drawnow.m: Check that canvas size is only specified
 	one time.
 
 	* plot/gnuplot_drawnow.m, plot/__go_draw_figure__.m: Support papersize,
@@ -211,17 +216,17 @@ 2009-02-20  Jaroslav Hajek  <highegg@gma
 
 	* linear-algebra/dmult.m: Remove.
 	* linear-algebra/Makefile.in: Update.
 	* deprecated/dmult.m: Move here; revert to the 3.0.x version.
 	* deprecated/Makefile.in: Update.
 	* statistics/models/logistic_regression_derivatives.m: Replace dmult
 	by diagonal matrices.
 
-2009-02-19  Ben Abbott <bpabbott@mac.com>
+2009-02-19  Ben Abbott  <bpabbott@mac.com>
 
 	* plot/gnuplot_drawnow.m: Only send figure position info to gnuplot
 	when the plot stream is first opened. Revert usage of hidden axes 
 	to store prior figure "position" property.
 
 2009-02-19  John W. Eaton  <jwe@octave.org>
 
 	* general/__isequal__.m, general/__splinen__.m, image/__img__.m,
diff --git a/scripts/plot/__go_draw_axes__.m b/scripts/plot/__go_draw_axes__.m
--- a/scripts/plot/__go_draw_axes__.m
+++ b/scripts/plot/__go_draw_axes__.m
@@ -1463,17 +1463,17 @@ function do_tics (obj, plot_stream, ymir
   if (strcmpi (obj.yaxislocation, "right"))
     do_tics_1 (obj.ytickmode, obj.ytick, obj.yminortick, obj.yticklabelmode,
 	       obj.yticklabel, obj.ycolor, "y2", plot_stream, ymirror, mono,
 	       "border", obj.tickdir, ticklength, fontname, fontsize,
 	       obj.interpreter);
     do_tics_1 ("manual", [], "off", obj.yticklabelmode, obj.yticklabel,
 	       obj.ycolor, "y", plot_stream, ymirror, mono, "border",
 	       "", "", fontname, fontsize, obj.interpreter);
-  elseif (strcmpi (obj.xaxislocation, "zero"))
+  elseif (strcmpi (obj.yaxislocation, "zero"))
     do_tics_1 (obj.ytickmode, obj.ytick, obj.yminortick, obj.yticklabelmode,
 	       obj.yticklabel, obj.ycolor, "y", plot_stream, ymirror, mono,
 	       "axis", obj.tickdir, ticklength, fontname, fontsize,
 	       obj.interpreter);
     do_tics_1 ("manual", [], "off", obj.yticklabelmode, obj.yticklabel,
 	       obj.ycolor, "y2", plot_stream, ymirror, mono, "axis",
 	       "", "", fontname, fontsize, obj.interpreter);
   else
